migrations/20210513172456-create-users-books.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("Users_Books", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      userId: {
+        allowNull: false,
+        references: {
+          model: "Users",
+          key: "id",
+        },
+        type: Sequelize.INTEGER,
+      },
+      bookId: {
+        allowNull: false,
+        references: {
+          model: "Books",
+          key: "id",
+        },
+        type: Sequelize.INTEGER,
+      },
```

Comment:
If you already have a relation between Books to Users, why do you need this table? Or is your idea is that users could share books? I mean, a book could have more than one user.